app/src/main/res/drawable/ic_search_40dp.xml

```diff
@@ -0,0 +1,6 @@
+<vector android:height="40dp" android:viewportHeight="310.42"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_search_40dp` appears to be unused
 Unused resources make applications larger and slow down builds.